src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -1,7 +1,5 @@
 /** @module FilterOptionsComponent */
 
-/* global Event */
```

Comment:
Should have been removed in https://github.com/yext/answers/pull/1058 to make semistandard happy, but circleci was not running my builds and I missed this